code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -9,8 +9,6 @@
 	health = 25
 	maxHealth = 25
 
-	radio_key = /obj/item/device/encryptionkey/headset_eng
-	radio_channel = "Engineering"
```

Comment:
Oh right I forgot the player floorbots. Nag functions and variables were made useless in the previous big change so they're not used in any way. But I'll add those radio things back for pAI bots.